doc/migrations/v0.42-v0.43.md

```diff
@@ -0,0 +1,79 @@
+# Migrating to libp2p@43 <!-- omit in toc -->
+
+A migration guide for refactoring your application code from libp2p v0.42.x to v0.43.0.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Circuit Relay v2](#circuit-relay-v2)
+
+## Circuit Relay v2
+
+libp2p@0.43.x ships with support for [Circuit Relay v2](https://github.com/libp2p/specs/blob/master/relay/circuit-v2.md).  This update to the spec changes the relay from being an open relay by default, and as such quite dangerous to enable, to being a limited relay with a slot reservation mechanism that is much safer.
```

Comment:
Does the JS impl use ASNs to limit the peers?

Reply:
Not out of the box, but the connection gater interface can be used to do this.